rna_cd/models.py

```diff
@@ -72,6 +72,10 @@ def train_svm_model(positive_bams: List[Path], negative_bams: List[Path],
         raise ValueError("The list of positive BAM files may not be empty.")
     if len(negative_bams) < 1:
         raise ValueError("The list of negative BAM files may not be empty.")
+
+    if set(positive_bams) == set(negative_bams):
+        raise ValueError("Positive and negative BAM files may not be "
+                         "identical.")
```

Comment:
This is not entirely the metric that you want I guess.
There may not be any negative bams that are also listed as positive bams and vice versa.
```python
if  not set(positive_bams).isdisjoint(set(negative_bams)):
    raise ValueError("bam files were given that occurred in both the positive and negative set")
# If you are nice you will also give them the list of these files using set's intersection method.
```
[`.isdisjoint`](https://docs.python.org/3.5/library/stdtypes.html#frozenset.isdisjoint) checks whether two sets have no elements in common.